### ISSUE TITLE
Error compiling openzeppelin dependencies

### ISSUE THREAD

**Issue by r4VP4** [ApeWorX/ape#662]
### Environment information

* OS: linux
* Python Version: 3.9.9
* `ape` and plugin versions:

```sh
$ ape --version
0.1.5

$ ape plugins list
Installed Plugins:
  alchemy     0.1.0
  solidity    0.1.1
  hardhat     0.1.4
```

* Contents of your `ape-config.yaml` (NOTE: do not post anything private like RPC urls or secrets!):

```sh
$ cat ape-config.yaml

name: ape_test1

dependencies:
  - name: OpenZeppelin
    github: OpenZeppelin/openzeppelin-contracts
    version: 4.5.0

solidity:
  import_remapping:
    - "@openzeppelin=OpenZeppelin/4.5.0"


hardhat:
  fork:
    ethereum:
      mainnet:
        upstream_provider: alchemy

```

### What went wrong?

compiling openzeppelin dependencies throws an error
(my contract-folder is empty, but i tried it with a dummy contract than i get an  error, too)
changing the openzeppelin version to 4.4.2 throws the same error

* what command you ran
ape compile / ape compile -f
* full output of the error you received
WARNING: 'OpenZeppelin' is not an 'ApeProject', but attempting to process as one.
INFO: Compiling 'mocks/ReentrancyAttack.sol'.
INFO: Compiling 'token/ERC1155/utils/ERC1155Holder.sol'.
INFO: Compiling 'token/ERC1155/IERC1155Receiver.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20VotesComp.sol'.
INFO: Compiling 'utils/introspection/ERC1820Implementer.sol'.
INFO: Compiling 'utils/math/Math.sol'.
INFO: Compiling 'mocks/AddressImpl.sol'.
INFO: Compiling 'utils/math/SignedMath.sol'.
INFO: Compiling 'token/ERC721/extensions/ERC721Pausable.sol'.
INFO: Compiling 'token/ERC1155/extensions/ERC1155Supply.sol'.
INFO: Compiling 'utils/introspection/IERC165.sol'.
INFO: Compiling 'mocks/ERC20CappedMock.sol'.
INFO: Compiling 'mocks/MathMock.sol'.
INFO: Compiling 'security/Pausable.sol'.
INFO: Compiling 'proxy/beacon/UpgradeableBeacon.sol'.
INFO: Compiling 'token/ERC721/ERC721.sol'.
INFO: Compiling 'utils/Base64.sol'.
INFO: Compiling 'mocks/DummyImplementation.sol'.
INFO: Compiling 'token/ERC721/presets/ERC721PresetMinterPauserAutoId.sol'.
INFO: Compiling 'mocks/GovernorTimelockCompoundMock.sol'.
INFO: Compiling 'mocks/ContextMock.sol'.
INFO: Compiling 'mocks/ERC721VotesMock.sol'.
INFO: Compiling 'proxy/beacon/IBeacon.sol'.
INFO: Compiling 'governance/IGovernor.sol'.
INFO: Compiling 'finance/PaymentSplitter.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Snapshot.sol'.
INFO: Compiling 'interfaces/IERC721Enumerable.sol'.
INFO: Compiling 'token/ERC777/IERC777.sol'.
INFO: Compiling 'proxy/ERC1967/ERC1967Proxy.sol'.
INFO: Compiling 'governance/utils/Votes.sol'.
INFO: Compiling 'mocks/ERC20FlashMintMock.sol'.
INFO: Compiling 'mocks/ERC777Mock.sol'.
INFO: Compiling 'mocks/GovernorVoteMock.sol'.
INFO: Compiling 'interfaces/IERC1155Receiver.sol'.
INFO: Compiling 'governance/extensions/GovernorSettings.sol'.
INFO: Compiling 'token/ERC20/ERC20.sol'.
INFO: Compiling 'mocks/ERC20PausableMock.sol'.
INFO: Compiling 'token/ERC721/IERC721.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Wrapper.sol'.
INFO: Compiling 'security/ReentrancyGuard.sol'.
INFO: Compiling 'interfaces/IERC777Recipient.sol'.
INFO: Compiling 'utils/escrow/Escrow.sol'.
INFO: Compiling 'mocks/ERC721RoyaltyMock.sol'.
INFO: Compiling 'mocks/ERC20BurnableMock.sol'.
INFO: Compiling 'mocks/BadBeacon.sol'.
INFO: Compiling 'mocks/Base64Mock.sol'.
INFO: Compiling 'mocks/ERC20VotesCompMock.sol'.
INFO: Compiling 'token/ERC721/utils/ERC721Holder.sol'.
INFO: Compiling 'token/ERC20/IERC20.sol'.
INFO: Compiling 'utils/escrow/RefundEscrow.sol'.
INFO: Compiling 'token/ERC721/extensions/IERC721Metadata.sol'.
INFO: Compiling 'mocks/SafeERC20Helper.sol'.
INFO: Compiling 'utils/cryptography/SignatureChecker.sol'.
INFO: Compiling 'interfaces/IERC3156.sol'.
INFO: Compiling 'mocks/ERC165/ERC165MissingData.sol'.
INFO: Compiling 'metatx/ERC2771Context.sol'.
INFO: Compiling 'utils/Multicall.sol'.
INFO: Compiling 'mocks/UUPS/UUPSLegacy.sol'.
INFO: Compiling 'interfaces/IERC2981.sol'.
INFO: Compiling 'mocks/GovernorCompMock.sol'.
INFO: Compiling 'mocks/MulticallTokenMock.sol'.
INFO: Compiling 'utils/Checkpoints.sol'.
INFO: Compiling 'mocks/EnumerableSetMock.sol'.
INFO: Compiling 'governance/extensions/GovernorVotesQuorumFraction.sol'.
INFO: Compiling 'mocks/SafeCastMock.sol'.
INFO: Compiling 'proxy/ERC1967/ERC1967Upgrade.sol'.
INFO: Compiling 'interfaces/IERC1155.sol'.
INFO: Compiling 'mocks/compound/CompTimelock.sol'.
INFO: Compiling 'mocks/SignedMathMock.sol'.
INFO: Compiling 'interfaces/IERC20Metadata.sol'.
INFO: Compiling 'interfaces/IERC721.sol'.
INFO: Compiling 'mocks/ERC721BurnableMock.sol'.
INFO: Compiling 'interfaces/IERC165.sol'.
INFO: Compiling 'mocks/EIP712External.sol'.
INFO: Compiling 'mocks/GovernorPreventLateQuorumMock.sol'.
INFO: Compiling 'utils/escrow/ConditionalEscrow.sol'.
INFO: Compiling 'governance/extensions/GovernorVotesComp.sol'.
INFO: Compiling 'interfaces/IERC1363.sol'.
INFO: Compiling 'utils/Counters.sol'.
INFO: Compiling 'mocks/CountersImpl.sol'.
INFO: Compiling 'token/ERC1155/utils/ERC1155Receiver.sol'.
INFO: Compiling 'mocks/ERC20VotesMock.sol'.
INFO: Compiling 'mocks/SignedSafeMathMock.sol'.
INFO: Compiling 'token/ERC20/extensions/IERC20Metadata.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Pausable.sol'.
INFO: Compiling 'utils/StorageSlot.sol'.
INFO: Compiling 'utils/structs/EnumerableMap.sol'.
INFO: Compiling 'token/ERC1155/extensions/ERC1155Burnable.sol'.
INFO: Compiling 'mocks/BitmapMock.sol'.
INFO: Compiling 'mocks/ERC3156FlashBorrowerMock.sol'.
INFO: Compiling 'token/ERC721/extensions/ERC721Royalty.sol'.
INFO: Compiling 'mocks/wizard/MyGovernor2.sol'.
INFO: Compiling 'interfaces/IERC1363Spender.sol'.
INFO: Compiling 'token/ERC20/presets/ERC20PresetMinterPauser.sol'.
INFO: Compiling 'utils/introspection/ERC165Checker.sol'.
INFO: Compiling 'utils/cryptography/ECDSA.sol'.
INFO: Compiling 'mocks/RegressionImplementation.sol'.
INFO: Compiling 'mocks/ERC1155ReceiverMock.sol'.
INFO: Compiling 'mocks/InitializableMock.sol'.
INFO: Compiling 'mocks/CallReceiverMock.sol'.
INFO: Compiling 'access/IAccessControl.sol'.
INFO: Compiling 'mocks/GovernorTimelockControlMock.sol'.
INFO: Compiling 'interfaces/IERC1155MetadataURI.sol'.
INFO: Compiling 'mocks/StringsMock.sol'.
INFO: Compiling 'mocks/Create2Impl.sol'.
INFO: Compiling 'access/AccessControlEnumerable.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Burnable.sol'.
INFO: Compiling 'token/ERC777/IERC777Sender.sol'.
INFO: Compiling 'governance/Governor.sol'.
INFO: Compiling 'mocks/ERC1155SupplyMock.sol'.
INFO: Compiling 'interfaces/IERC3156FlashLender.sol'.
INFO: Compiling 'mocks/ERC721URIStorageMock.sol'.
INFO: Compiling 'token/ERC777/IERC777Recipient.sol'.
INFO: Compiling 'mocks/ClonesMock.sol'.
INFO: Compiling 'governance/TimelockController.sol'.
INFO: Compiling 'token/ERC721/extensions/ERC721Burnable.sol'.
INFO: Compiling 'utils/introspection/IERC1820Implementer.sol'.
INFO: Compiling 'governance/utils/IVotes.sol'.
INFO: Compiling 'token/ERC721/IERC721Receiver.sol'.
INFO: Compiling 'token/ERC721/extensions/IERC721Enumerable.sol'.
INFO: Compiling 'mocks/ERC165StorageMock.sol'.
INFO: Compiling 'utils/structs/EnumerableSet.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Votes.sol'.
INFO: Compiling 'mocks/AccessControlEnumerableMock.sol'.
INFO: Compiling 'mocks/PullPaymentMock.sol'.
INFO: Compiling 'mocks/ReentrancyMock.sol'.
INFO: Compiling 'utils/structs/BitMaps.sol'.
INFO: Compiling 'governance/compatibility/IGovernorCompatibilityBravo.sol'.
INFO: Compiling 'mocks/PausableMock.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20FlashMint.sol'.
INFO: Compiling 'governance/extensions/GovernorVotes.sol'.
INFO: Compiling 'mocks/ClashingImplementation.sol'.
INFO: Compiling 'utils/math/SafeMath.sol'.
INFO: Compiling 'governance/extensions/IGovernorTimelock.sol'.
INFO: Compiling 'mocks/SingleInheritanceInitializableMocks.sol'.
INFO: Compiling 'utils/Arrays.sol'.
INFO: Compiling 'proxy/utils/Initializable.sol'.
INFO: Compiling 'utils/math/SafeCast.sol'.
INFO: Compiling 'utils/math/SignedSafeMath.sol'.
INFO: Compiling 'mocks/VotesMock.sol'.
INFO: Compiling 'interfaces/IERC777Sender.sol'.
INFO: Compiling 'mocks/ERC1155BurnableMock.sol'.
INFO: Compiling 'mocks/ERC777SenderRecipientMock.sol'.
INFO: Compiling 'mocks/SafeMathMock.sol'.
INFO: Compiling 'mocks/ERC20PermitMock.sol'.
INFO: Compiling 'token/ERC20/extensions/ERC20Capped.sol'.
INFO: Compiling 'token/ERC777/ERC777.sol'.
INFO: Compiling 'mocks/GovernorCompatibilityBravoMock.sol'.
INFO: Compiling 'token/ERC20/utils/TokenTimelock.sol'.
INFO: Compiling 'interfaces/IERC20.sol'.
INFO: Compiling 'mocks/ConditionalEscrowMock.sol'.
INFO: Compiling 'interfaces/IERC721Receiver.sol'.
INFO: Compiling 'utils/Strings.sol'.
INFO: Compiling 'mocks/OwnableMock.sol'.
INFO: Compiling 'security/PullPayment.sol'.
INFO: Compiling 'finance/VestingWallet.sol'.
INFO: Compiling 'utils/introspection/ERC165Storage.sol'.
INFO: Compiling 'access/IAccessControlEnumerable.sol'.
INFO: Compiling 'access/Ownable.sol'.
INFO: Compiling 'governance/extensions/GovernorTimelockControl.sol'.
INFO: Compiling 'utils/Timers.sol'.
INFO: Compiling 'mocks/ERC721EnumerableMock.sol'.
INFO: Compiling 'governance/extensions/GovernorProposalThreshold.sol'.
INFO: Compiling 'interfaces/IERC3156FlashBorrower.sol'.
INFO: Compiling 'mocks/ERC1820ImplementerMock.sol'.
INFO: Compiling 'token/ERC777/presets/ERC777PresetFixedSupply.sol'.
INFO: Compiling 'proxy/Proxy.sol'.
INFO: Compiling 'token/ERC20/presets/ERC20PresetFixedSupply.sol'.
INFO: Compiling 'mocks/AccessControlMock.sol'.
INFO: Compiling 'proxy/Clones.sol'.
INFO: Compiling 'mocks/EtherReceiverMock.sol'.
INFO: Compiling 'utils/Address.sol'.
INFO: Compiling 'mocks/ArraysImpl.sol'.
INFO: Compiling 'interfaces/IERC1363Receiver.sol'.
INFO: Compiling 'utils/Context.sol'.
INFO: Compiling 'token/ERC721/extensions/ERC721Enumerable.sol'.
INFO: Compiling 'mocks/MerkleProofWrapper.sol'.
INFO: Compiling 'mocks/ERC2771ContextMock.sol'.
INFO: Compiling 'proxy/transparent/ProxyAdmin.sol'.
INFO: Compiling 'proxy/utils/UUPSUpgradeable.sol'.
INFO: Compiling 'mocks/ERC165CheckerMock.sol'.
INFO: Compiling 'mocks/ERC721ReceiverMock.sol'.
INFO: Compiling 'mocks/TimersBlockNumberImpl.sol'.
INFO: Compiling 'mocks/GovernorMock.sol'.
INFO: Compiling 'mocks/TimersTimestampImpl.sol'.
INFO: Compiling 'interfaces/IERC721Metadata.sol'.
INFO: Compiling 'token/ERC1155/ERC1155.sol'.
INFO: Compiling 'mocks/ERC721Mock.sol'.
INFO: Compiling 'interfaces/IERC777.sol'.
INFO: Compiling 'mocks/ERC20WrapperMock.sol'.
INFO: Compiling 'mocks/ERC165/ERC165NotSupported.sol'.
INFO: Compiling 'mocks/ECDSAMock.sol'.
INFO: Compiling 'mocks/UUPS/UUPSUpgradeableMock.sol'.
INFO: Compiling 'token/ERC20/utils/SafeERC20.sol'.
INFO: Compiling 'token/ERC1155/extensions/IERC1155MetadataURI.sol'.
INFO: Compiling 'mocks/CheckpointsImpl.sol'.
INFO: Compiling 'mocks/wizard/MyGovernor3.sol'.
INFO: Compiling 'interfaces/IERC1820Implementer.sol'.
INFO: Compiling 'token/ERC1155/extensions/ERC1155Pausable.sol'.
INFO: Compiling 'utils/cryptography/MerkleProof.sol'.
INFO: Compiling 'token/ERC1155/IERC1155.sol'.
INFO: Compiling 'proxy/beacon/BeaconProxy.sol'.
INFO: Compiling 'governance/extensions/GovernorCountingSimple.sol'.
INFO: Compiling 'mocks/ERC1271WalletMock.sol'.
INFO: Compiling 'token/common/ERC2981.sol'.
INFO: Compiling 'mocks/MultipleInheritanceInitializableMocks.sol'.
INFO: Compiling 'metatx/MinimalForwarder.sol'.
INFO: Compiling 'token/ERC1155/presets/ERC1155PresetMinterPauser.sol'.
INFO: Compiling 'utils/introspection/IERC1820Registry.sol'.
INFO: Compiling 'proxy/transparent/TransparentUpgradeableProxy.sol'.
INFO: Compiling 'mocks/ERC20DecimalsMock.sol'.
INFO: Compiling 'mocks/ERC165Mock.sol'.
INFO: Compiling 'governance/extensions/GovernorTimelockCompound.sol'.
INFO: Compiling 'interfaces/IERC1271.sol'.
INFO: Compiling 'mocks/wizard/MyGovernor1.sol'.
INFO: Compiling 'utils/Create2.sol'.
INFO: Compiling 'mocks/ERC20SnapshotMock.sol'.
INFO: Compiling 'mocks/ERC20Mock.sol'.
INFO: Compiling 'interfaces/IERC1820Registry.sol'.
INFO: Compiling 'mocks/ERC1155PausableMock.sol'.
INFO: Compiling 'mocks/SignatureCheckerMock.sol'.
INFO: Compiling 'mocks/StorageSlotMock.sol'.
INFO: Compiling 'mocks/ERC165/ERC165InterfacesSupported.sol'.
INFO: Compiling 'mocks/ERC1155Mock.sol'.
INFO: Compiling 'token/ERC721/extensions/ERC721URIStorage.sol'.
INFO: Compiling 'mocks/EnumerableMapMock.sol'.
INFO: Compiling 'mocks/MulticallTest.sol'.
INFO: Compiling 'utils/introspection/ERC165.sol'.
INFO: Compiling 'governance/extensions/GovernorPreventLateQuorum.sol'.
INFO: Compiling 'access/AccessControl.sol'.
INFO: Compiling 'governance/compatibility/GovernorCompatibilityBravo.sol'.
INFO: Compiling 'mocks/ERC721PausableMock.sol'.
Traceback (most recent call last):
  File "/home/tw/.pyenv/versions/ape1/bin/ape", line 8, in <module>
    sys.exit(cli())
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/_cli.py", line 40, in invoke
    return super().invoke(ctx)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape_compile/_cli.py", line 53, in cli
    contract_types = cli_ctx.project_manager.load_contracts(
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/managers/project/manager.py", line 393, in load_contracts
    self._load_dependencies()
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/managers/project/manager.py", line 403, in _load_dependencies
    api.extract_manifest()  # Downloads if needed
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/managers/project/dependency.py", line 97, in extract_manifest
    return self._extract_local_manifest(temp_project_path)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/api/projects.py", line 233, in _extract_local_manifest
    project_manifest = project.create_manifest(file_paths=sources)
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/managers/project/types.py", line 147, in create_manifest
    manifest = self._create_manifest(
  File "/home/tw/.pyenv/versions/3.9.9/envs/ape1/lib/python3.9/site-packages/ape/api/projects.py", line 119, in _create_manifest
    return PackageManifest(**manifest.dict())
  File "pydantic/main.py", line 331, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for PackageManifest
__root__
  'interfaces/draft-IERC1822.sol' missing from `sources` (type=value_error)

**Comment by fubuloubu** [comment]
I've been getting this too lately. No idea why though, v4.5.0 is several months old, it must be a regression in Ape v0.1.5

Can you try it downgraded to Ape v0.1.4?

**Comment by fubuloubu** [comment]
@r4VP4 should be fixed as of v0.1.6

**Comment by r4VP4** [comment]
upgraded to v0.1.6: everything's fine! Appreciate your fast fixing! Thank you very much!
